tests/pytests/unit/test_master.py

```diff
@@ -4,7 +4,37 @@
 
 import salt.master
 import salt.utils.platform
-from tests.support.mock import patch
+from tests.support.mock import MagicMock, patch
+
+
+@pytest.fixture
+def maintenence_opts(master_opts):
+    """
+    Options needed for master's Maintenence class
+    """
+    opts = master_opts.copy()
+    opts.update(git_pillar_update_interval=180, maintenance_interval=181)
```

Comment:
magic numbers, understand it is a test

Reply:
Yeah, and a test that existed before this PR. I'm simply migrating it to pytest. I don't want to spend time fiddling with the numbers unless the test is flaky or doesn't work.